bundles/org.openhab.binding.pentair/src/main/resources/OH-INF/thing/controller.xml

```diff
@@ -0,0 +1,375 @@
+			</channel-group>
+
+			<channel-group id="schedule9" typeId="scheduletype">
+				<label>Schedule 9</label>
+			</channel-group>
+
+			<channel-group id="status" typeId="statustype"/>
+		</channel-groups>
+
+		<properties>
+			<property name="fwversion">FWVersion</property>
+		</properties>
+
+		<representation-property>id</representation-property>
+
+		<config-description>
+			<parameter name="id" type="integer">
+				<label>ID</label>
+				<description>The ID of the device (in decimal, not hex)</description>
+				<default>16</default>
+			</parameter>
+
+			<parameter name="synctime" type="boolean">
+				<label>Synchronize Time</label>
+				<description>Enables automatic synchronization of the pool controller clock with the system clock</description>
+				<default>true</default>
+			</parameter>
+
+		</config-description>
+
+	</thing-type>
+
+	<channel-group-type id="circuitgrouptype">
+		<label>Circuit</label>
+		<description>Circuit</description>
+		<channels>
+			<channel id="switch" typeId="auxswitchtype"/>
+			<channel id="name" typeId="circuitnametype"/>
+			<channel id="function" typeId="circuitfunctiontype"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="featuretype">
+		<label>Feature</label>
+		<description>Features</description>
+		<channels>
+			<channel id="switch" typeId="auxswitchtype"/>
+			<channel id="name" typeId="circuitnametype"/>
+			<channel id="function" typeId="circuitfunctiontype"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="heattype">
+		<label>Heat</label>
+		<description>Heat</description>
+		<channels>
+			<channel id="heatmode" typeId="heatmodetype"/>
+			<channel id="setpoint" typeId="setpointtype"/>
+			<channel id="temperature" typeId="temperaturetype"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="scheduletype">
+		<label>Schedule</label>
+		<description>schedule</description>
+		<channels>
+			<channel id="schedule" typeId="schedulestringtype"/>
+			<channel id="type" typeId="scheduletype"/>
+			<channel id="start" typeId="starttype"/>
+			<channel id="end" typeId="endtype"/>
+			<channel id="circuit" typeId="circuittype"/>
+			<channel id="days" typeId="daystype"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="statustype">
+		<label>Status</label>
+		<description>General status channels for controller</description>
+		<channels>
+			<channel id="lightmode" typeId="lightmodetype"/>
+			<channel id="solartemperature" typeId="solartemptype"/>
+			<channel id="airtemperature" typeId="airtemptype"/>
+			<channel id="heaterdelay" typeId="heaterdelaytype"/>
+			<channel id="uom" typeId="uomtype"/>
+			<channel id="servicemode" typeId="servicemodetype"/>
+			<channel id="solaron" typeId="solarontype"/>
+			<channel id="heateron" typeId="heaterontype"/>
+		</channels>
+
+	</channel-group-type>
+
+	<channel-type id="temperaturetype">
+		<item-type>Number:Temperature</item-type>
+		<label>Water temperature</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
sorry, i thought I fixed all of those already :(